examples/coexist_klog_v1/coexist_klog_v1.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogWriter struct {}
+func (kw klogWriter)Write(p []byte) (n int, err error){
```

Comment:
there should be a space before `Write`.
the github actions / gofmt should be catching this, but it doesn't seem like so.

Reply:
i don't know how this linter works, but it seems we are telling it to do gofmt:
https://github.com/kubernetes/klog/blob/master/.github/workflows/test.yml#L32

Reply:
Fixed.